### ISSUE TITLE
Wallet - Implement asset actions bottom sheet UI in Wallet home

### ISSUE THREAD

**Issue by smohamedjavid** [status-im/status-react#18420]
# Feature Issue

Implement asset actions bottom sheet UI in Wallet home

<img width="500"  src="https://github.com/status-im/status-mobile/assets/19339952/04c977f7-4ddb-4e60-a2f3-eee2ea1c426a">

### Description

On long press of an asset displayed, the bottom sheet should be displayed with actions `Buy`, `Send`, `Receive`, `Bridge`, `Manage tokens`, and `Hide`.

### Acceptance Criteria

- On long press of asset, the bottom sheet is displayed
- The actions `Buy`, `Send`, `Receive`, `Bridge`, `Manage tokens`, and `Hide` are displayed

### Notes

This issue is a UI-only implementation. The integration of each action will be handled later in a separate issue

**Comment by J-Son89** [comment]
please align with design before starting this as some steps are missing in designs, e.g which account are you sending from etc :)

**Comment by smohamedjavid** [comment]
Closing this issue until the requirements are cleared out.
